### DISCUSSION TITLE
Feature Suggestion: Toggle for User Sounds Area in CS Demo Manager

### DISCUSSION THREAD

**Discussion by grok-rs** [akiver/cs-demo-manager#946] [Ideas]
Hi @akiver 

First of all, thank you for creating and maintaining CS Demo Manager! It’s an incredible tool that has greatly helped the community analyze and learn from Counter-Strike games.

I wanted to suggest a potential feature that could be useful in the context of CS2 demos, especially considering the new sound mechanics. In CS2, players can now visually see the area of sound on the minimap (e.g., footsteps that other players can hear). I believe adding a toggle option in CS Demo Manager to display this user sound area would be extremely beneficial for players who want to analyze how their movement and sounds affect other players' awareness.

Feature Request:
A toggle to show or hide the user-generated sound area in demos.
This feature could visualize when and where steps or other sounds (like gun reloads, jumps, etc.) are being heard by opponents.
It would allow users to analyze their movements and whether they were heard by enemies in a more detailed and precise manner.
I think this feature would greatly enhance the usefulness of CS Demo Manager for in-depth analysis of game demos.

Do you think it would be possible to integrate something like this?

Thank you again for all your hard work on this tool. I look forward to hearing your thoughts!

**Comment by akiver** [comment]
Hi!

That would be a cool addition.

Not 100% sure but it looks like the sound radius in units of weapons, player movements... can be found in `soundevents/game_sounds_xxx.vsndevts_c` (the `distance_volume_mapping_curve` field)
